appinventor/docs/reference/components/connectivity.html

```diff
@@ -550,7 +550,7 @@ <h3>Properties</h3>
   <dt><code>HighByteFirst</code></dt>
   <dd></dd>
   <dt><code><em>IsConnected</em></code></dt>
-  <dd></dd>
+  <dd>On devices with API level 14 (LEVEL_ICE_CREAM_SANDWICH) or higher, this property returned is accurate. But on old devices with API level lower than 14, it may not return the current state of connection(e.g., it might be disconnected but you may not know until you attempt to read/write the socket.)</dd>
```

Comment:
Now that we have markdown based documentation, you should include this in the description in the `@SimpleProperty` annotation as this file no longer exists.